network/src/services/outbound_peer.rs

```diff
@@ -155,14 +155,20 @@ impl Future for OutboundPeerService {
     fn poll(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Self::Output> {
```

Comment:
This PR splited the implementation of `DumpPeerStoreService->poll`, but does't split the implementation of `OutboundPeerService->poll` for `wasm` and `not(wasm)`. Is this expected?

Reply:
yes, it seems github ci win/macos has some issue on disk，if the DumpPeerStoreService poll doesn't split with `wasm` and `not(wasm)`, it will fail on ci.

The difference between their implementations is whether to perform a dump operation immediately.